### DISCUSSION TITLE
The Mod Manager can softlock the game

### DISCUSSION THREAD

**Discussion by BigBoss320** [Raytwo/ARCropolis#359] [Issues]
This is not important but i noticed that the game can be softlock when we are in the mod manager, then go to the home menu (without exiting the game) then return to the game. This also do it when power off and on the switch (by briefly pressing once on the power button)

**Comment by Raytwo** [comment]
Would blocking usage of the home button while the mod manager is open be acceptable as a solution?

**Reply by BigBoss320** [reply]
Yeah I think it is the best solution (if it's possible to do that).
More precisely it happens when you open the album in Homebrew or Normal mod ( using R button) 
I just tested that now.
